typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transcend-io/typescript-webhook-example",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "description": "Example of a webhook that can be integrated with Transcend.",
```

Comment:
you will also have to update `"files": [` in the package.json for this to work. not sure if its possible to include something in a parent directory tho.... may need to copy this folder into this package maybe

Reply:
ah!

Reply:
There you go!